src/objective-c/GRPCClient/private/NSDictionary+GRPC.h

```diff
@@ -32,9 +32,9 @@
  */
 
 #import <Foundation/Foundation.h>
-
-struct grpc_metadata;
+#include <grpc/grpc.h>
```

Comment:
It's simpler indeed. The downside is it makes compilation slower, and in some dependency cases inclusion doesn't work and forward declaration is needed. But as we're layering on top of that library, inclusion might be better.